### PR TITLE
Style closed petition view

### PR DESCRIPTION
**Pull request by henryhadlow**
* Make buttons normal width on tablet/desktop
* Make a generic .secondary grey-text class
* Remove 'How petitions work' from petitions show
* Add new closed_petition_show view
* Move _petition_show to _open_petition_show and remove conditional logic
* Typography tweaks

### CODE REVIEW

**Comment by henryhadlow** [comment]
Hold it! Forgot to delete a redundant view

**Comment by h-lame** [comment]
Assuming this goes green, :+1: from me.

File: app/views/petitions/_open_petition_show.html.erb [hunk 1]
```diff
@@ -15,10 +11,22 @@
   </details>
 <% end %>
 
-<%# Signable #%>
-<% if petition.can_be_signed? -%>
```
Comment (alexander-bobin) [inline app/views/petitions/_open_petition_show.html.erb]: Looks like `can_be_signed` is only being used in one other place now. It shouldn't be changed in this PR but we could probably drop it

**Comment by alexander-bobin** [comment]
All looks OK to me accept I think the story needs some ACs in Pivotal so Alan can test it

**Pull request merged**